client/app/assets/styles/variables.js

```diff
@@ -111,6 +111,7 @@ module.exports = {
   '--Topbar_tabletHeight': '60px',
   '--Topbar_mobileHeight': '50px',
   '--Topbar_padding': `0 ${bodyPadding}`,
+  '--Topbar_paddingLanguageMenuVertical': '27.5px',
```

Comment:
Language menu's font-size (13px) + 2 * --Topbar_paddingLanguageMenuVertical must be 70px (to be on same level as other dropdowns).